MfciPkg/UnitTests/MfciPolicyParsingUnitTest/data/certs/MakeChainingCerts.bat

```diff
@@ -1,3 +1,9 @@
+REM Copyright (c) Microsoft Corporation.

```

Comment:
should this file just be deleted?

Reply:
Windows 11 SDK (10.0.22621.0) (latest Windows 11 SDK) still contains the MakeCert utility.  It is the MakeCert page on learn.microsoft.com that specifies that MakeCert is deprecated. (https://learn.microsoft.com/en-us/windows/win32/seccrypto/makecert)

I don't have a problem leaving the original file available as long as the tool is still being distributed, but I also am not opposed to removing it.